spec/index.bs

```diff
@@ -9728,6 +9741,7 @@ called the render pass encoder can no longer be used.
                         - |this| must be [=valid=].
                         - |this|.{{GPUDebugCommandsMixin/[[debug_group_stack]]}} must [=list/is empty|be empty=].
                         - |this|.{{GPURenderPassEncoder/[[occlusion_query_active]]}} must be `false`.
+                        - |this|.{{GPURenderCommandsMixin/[[drawCount]]}} must be less than |this|.{{GPURenderPassEncoder/[[maxDrawCount]]}}.
```

Comment:
(Sorry for adding a comment to a merged PR.)

Shouldn't drawCount == maxDrawCount be valid? If so, this line should be replaced with

> this.[[drawCount]] must be equal to or less that this.[[maxDrawCount]].

I want to make a PR if it sounds ok.

Reply:
Good catch! Can you make a PR to change this? ("must be less or equal" is more idiomatic I think? Though English is not my native language).

Reply:
English isn't my first language, neither, so I googled.

"less than or equal to" - About 60,400,000 results

"equal to or less than" - About 24,400,000 results

"less than or equal to" may be more natural.